frontend/src/components/DiscoveryDetails.js

```diff
@@ -147,7 +148,7 @@ const StyledNotesSection = styled.div`
 display: block;
 grid-column-start: 1;
 grid-column-end: 2;
```

Comment:
To get back to that topic, you could shorten it here to grid-column: 1 /2; :-)

Reply:
Great tip, thank you Saskia!